CoffeeShop.playground/Contents.swift

```diff
@@ -0,0 +1,413 @@
+import Foundation
+
+// MARK: - Location
+struct Location: Equatable {
+    var x = 0.0
+    var y = 0.0
+
+    func distanceTo(_ location: Location) -> Double {
+        return sqrt( pow(self.x - location.x, 2) + pow(self.y - location.y, 2) )
+    }
+}
+
+
+typealias CoffeeShopWithDistance = (coffeeShop: CoffeeShop, distance: Double)
+
+// MARK: - CoffeeShop
+struct CoffeeShop: Equatable {
+
+    var name: String
+    var location: Location
+
+    init(name: String, location: Location) {
+        self.name = name
+        self.location = location
+    }
+
+    init?(_ line: String, separator: String) {
```

Comment:
What is the `_` used for here ?

Reply:
It means that the label of the argument is missing 
https://docs.swift.org/swift-book/LanguageGuide/Functions.html#ID166